theseus/geometry/se3.py

```diff
@@ -16,6 +16,9 @@
 
 
 class SE3(LieGroup):
+    NEAR_PI_EPS = 1e-7
+    NEAR_ZERO_EPS = 5e-3
```

Comment:
I did a quick search before making the change and noticed the same value was used everywhere, so I figured I would keep one value for now. The reason I moved it here is because I split `_log_map_impl()` into two functions that were using the same EPS variable. 

In any case, we can probably defer this until we do more testing to see what the right EPS is for each function, and refactor accordingly.